extensions/Cake.DocCreator.yml

```diff
@@ -3,7 +3,7 @@ Name: Cake.DocCreator
 NuGet: Cake.DocCreator
 Assemblies:
 - "/**/Cake.DocCreator.dll"
-Repository: https://bitbucket.org/agc93/DocCreator/
```

Comment:
The bitbucket URL is invalid:

![image](https://user-images.githubusercontent.com/112710/101081226-cce6f300-3577-11eb-947b-58e643bf5f1c.png)

URL validation is super slow so I turn it off when I run AddinDiscoverer on my laptop (like I did this morning to update all the YAML files) that explains why the YAML we generated earlier today for this addin contained an invalid URL. After you and I were done updating all YAML files, I published a new version of the AddinDisco with the new improvements and therefore it ran with the URL validation turned on which caused the invalid URL to be detected.